Taarafo.Core.Tests.Unit/Services/Foundations/PostImpressions/PostImpressionServiceTests.Exceptions.Add.cs

```diff
@@ -39,9 +39,13 @@ public async Task ShouldThrowCriticalDependencyExceptionOnAddIfSqlErrorOccursAnd
             ValueTask<PostImpression> addPostImpressionTask =
                 this.postImpressionService.AddPostImpressions(somePostImpression);
 
+            PostImpressionDependencyException actulaPostImpressionDependencyException =
```

Comment:
there is a typo in the word `actulaPostImpressionDependencyException`
instead of `actula`, it should be `actual`